cmd/zar/zq/command.go

```diff
@@ -149,37 +133,11 @@ func (c *Command) Run(args []string) error {
 	})
 }
 
-func (c *Command) verifyPaths(paths []string) ([]string, error) {
```

Comment:
Would you explain why you dropped the `verifyPaths` logic?

Reply:
First off it seemed excessive to do an extra network hop when the call to read the data would give us the same information. Secondly looking at the code below it looks like we are wrapping the created reader in a warningsReader meaning that we'll get the same error message anyways.